templates/web/pages/bmviii_home.tmpl

```diff
@@ -0,0 +1,342 @@
+                        <div class="bmviii-right-accordion-content">
+                            <p> style="font-family: Cabin;"
+                                Feel free to email team@boilermake.org for info about our waitlist, and we will do our best to accommodate you as space opens up at our event.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="container bmviii-container-accordion">
+                        <button class="bmviii-right-accordion">Do you allow walk-ins and last minute registration?</button>
+                        <div class="bmviii-right-accordion-content">
+                            <p style="font-family: Cabin;">
+                                We accept walk-ins based on availability after hacker check-in.
+                                We will announce more details closer to the event!
+                            </p>
+                        </div>
+                    </div>
+                    <div class="container bmviii-container-accordion">
+                        <button class="bmviii-right-accordion">I missed the deadline to apply - can I attend?</button>
+                        <div class="bmviii-right-accordion-content">
+                            <p style="font-family: Cabin;">
+                                We set a deadline to ensure that all applications are given equal consideration to attend BoilerMake.
+                                Feel free to email team@boilermake.org for info about our waitlist, and we will do our best to accommodate you as space opens up at our event.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="container">
+                        <button class="bmviii-right-accordion">What if I have any other questions?</button>
+                        <div class="bmviii-right-accordion-content">
+                            <p style="font-family: Cabin;">
+                                Please email us at <a href='mailto:team@boilermake.org'>team@boilermake.org</a>!
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container" style="padding: 3rem 1.5rem;" >
+        <div class="container" style="padding: 1rem 1.5rem;">
+                <div class="columns is-mobile is-centered content">
+                    <div class="column is-6-mobile is-4-tablet is-4-desktop" style="margin-left: 48px; margin-right: 12px; margin-top:72px">
+                        <img alt="Testimonial" src={{ static_path "bmviii_testimonial_text.svg" }}>
+                    </div>
+                </div>
+        </div>
+        <div class="carousel-wrapper" style="margin-top: 125px; margin-left: 1.5rem">
```

Comment:
![Screen Shot 2020-10-30 at 21 03 19](https://user-images.githubusercontent.com/10781907/97767575-f5984a80-1aea-11eb-8333-5793b8e75ab4.png)
The way this is set up on the towel looks kinda awkward, the mix of the square corners on the image and then the round corners of the text blurb is a little weird. Maybe something like this would be a little better?
![Screen Shot 2020-10-30 at 21 05 41](https://user-images.githubusercontent.com/10781907/97767631-4ad45c00-1aeb-11eb-8309-de3d3d6d64ac.png)

Even better might be if you could have the image fill the whole left half and get kind of clipped by the rounded corners.

Reply:
I'm not sure how to fill the image, but I can round the corners using the border-radius property

![Screen Shot 2020-10-30 at 10 23 07 PM](https://user-images.githubusercontent.com/31838229/97769115-99d7bc80-1afe-11eb-8796-b6daf0aebfdf.png)

Reply:
Yeah I think that def looks better